### ISSUE TITLE
FSR mod in SteamVR/IL2 no longer works.

### ISSUE THREAD

**Issue by Panzerlang** [fholger/openvr_fsr#143]
Hi.
Since a few days ago (and couple of weeks ago for the Quest2) the FSR API for the G2 has not only stopped working in IL2, it has trashed the graphics quality. Reverting to IL2's stock API fixes it but obviously no longer with the FSR mod goodness. I've tried old NVidia drivers, no joy, so clearly Steam have busted it with their latest SteamVR update (and ditto facebook with the Quest2).

It would be so appreciated if FSR could be updated to fix this, the difference it made in IL2 was almost miraculous.

TIA and kind regards. :)

**Comment by fholger** [comment]
I'm confused. You mentioned in the other thread that IL2 uses OpenXR. If that's the case, the mod could never have worked in the first place. Or if that's a recent switch, then that would explain why it's no longer working.

**Comment by Panzerlang** [comment]
> I'm confused. You mentioned in the other thread that IL2 uses OpenXR. If that's the case, the mod could never have worked in the first place. Or if that's a recent switch, then that would explain why it's no longer working.

Yeah, sorry about that. It's confusing because the WMR window has a message saying how OpenXR isn't supported and might cause issues. If you then choose to "Fix It" it makes the runtime go to WMR instead of...you guessed it...Steam's OpenVR (NOT OpenXR). I can only surmise (as I frequently do) that one or both of WMR and SteamVR are run and operated by absolute incompetents.
So, anyway, they (Oculus AND Steam) have done something to stop FSR from working in IL2 now.

**Comment by Panzerlang** [comment]
I should add that it's 100% not a game-update thing and I've tried the older NVidia 172 drivers, no joy. Hand in hand with the FSR failure was a whole bunch of other SteamVR screwups that took me around three hours to work out until I could get IL2 running in VR again, so I'm very certain it's a SteamVR thing.

**Comment by Panzerlang** [comment]
So...it isn't broken, it was an in-game setting that I swear I never changed but must have I guess. Old age, the curse.

**Comment by Piange** [comment]
Hi @Panzerlang ! Can you please explain what was the in-game setting ? (i'm an il2 player too)
